playwright.config.js

```
@@ -0,0 +1,26 @@
+const { defineConfig, devices } = require("@playwright/test");
```

ESLint

[error] Replace `"`@playwright/test`"` with `'`@playwright/test`'`

(prettier/prettier)